README.md

```diff
@@ -5,7 +5,7 @@ Blueprint is a meta-build system that reads in Blueprints files that describe
 modules that need to be built, and produces a Ninja
 (http://martine.github.io/ninja/) manifest describing the commands that need to
 be run and their dependencies.  Where most build systems use built-in rules or
-a domain-specific langauge to describe the logic for converting module
+a domain-specific language to describe the logic for converting module
```

Comment:
The same typo is in doc.go, can you fix it there too?

Also, we'll need a signed CLA to accept the patch, see the procedure in the not-yet-merged file https://github.com/colincross/blueprint/blob/ed5c21b1dc7d20290d2dccfd14eaf29adf7741b6/CONTRIBUTING.md